doc/developer/lua.rst

```diff
@@ -0,0 +1,59 @@
+LUA
+===
+
+LUA is currently experimental within FRR and has very limited
+support.  If you would like to compile FRR with LUA you must
+follow these steps
```

Comment:
`:` is missing at the end (after steps) or at least `.`

Reply:
fixed